### PR TITLE
#1400. Composing inline classes tests

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

**Review by eernstg — COMMENTED** [review]
Looks very good, just a couple of comments.

File: LanguageFeatures/Inline-classes/composing_inline_classes_A05_t01.dart [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  V1(this.id);
+
+  int m() => 42;
+}
+
+inline class V2 {
+  final int id;
+  V2(this.id);
+
+  String m() => "42";
```
[the remark this answers is not in the document]
Comment (sgrekhov) [reply LanguageFeatures/Inline-classes/composing_inline_classes_A05_t01.dart:34]: Added a separate tests `composing_inline_classes_A05_t05.dart`, testing equal signatures case (I'll renumerate tests in a separate commit to make a new test to follow `composing_inline_classes_A05_t01.dart`)

File: LanguageFeatures/Inline-classes/composing_inline_classes_A05_t02.dart [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  V1(this.id);
+
+  int m() => 1;
+}
+
+inline class V2 {
+  final int id;
+  V2(this.id);
+
+  int m() => 2;
```
[the remark this answers is not in the document]
Comment (sgrekhov) [reply LanguageFeatures/Inline-classes/composing_inline_classes_A05_t02.dart:33]: Yes, right

File: LanguageFeatures/Inline-classes/composing_inline_classes_A05_t04.dart [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+/// say that V1 is a superinterface of DV.
+/// ...
+/// A compile-time error occurs if an inline class declaration DV has two
+/// superinterfaces V1 and V2, where both V1 and V2 have a member named m, and
+/// the two declarations of m are distinct declarations, and DV does not declare
+/// a member named m.
+///
+/// @description Checks that it is no an error if an inline class declaration
+/// `DV` has two superinterfaces `V1` and `V2`, where both `V1` and `V2` have
+/// the same member named `m`
```
[the remark this answers is not in the document]
Comment (sgrekhov) [reply LanguageFeatures/Inline-classes/composing_inline_classes_A05_t04.dart]: Changed "same member" to "same declaration"

**Review by sgrekhov — COMMENTED** [review]
Updated. Please rereview

File: LanguageFeatures/Inline-classes/composing_inline_classes_A05_t01.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (eernstg) [reply LanguageFeatures/Inline-classes/composing_inline_classes_A05_t01.dart:34]: Sounds good!

File: LanguageFeatures/Inline-classes/composing_inline_classes_A05_t04.dart (hunk 3 above)
[the remark this answers is not in the document]
Comment (eernstg) [reply LanguageFeatures/Inline-classes/composing_inline_classes_A05_t04.dart]: Good, thanks!

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**